docs/cli.md

```diff
@@ -119,6 +121,27 @@ device-3: localhost-3 (1.0.0)
 device-4: localhost-4 (1.0.0)
 ```
 
+### Listing and Loading model plugins
+A model plugin is a shared object library that represents the YANG models of a
+particular Device Type and Version. The plugin allows user to create and load
+their own device models in to onos-config that can be used for validating that
+configuration changes observe the structure of the YANG models in use on the
+device. This improves usability by pushing information about the devices'
+model back up to the onos-config gNMI northbound interface.
+
+Model plugins can be loaded at the startup of onos-config by (repeated) --modelPlugin
+options, or they can be loaded at run time with the command:
+```bash
```

Comment:
Text above implies that what's coming is a load command, not list command.

Reply:
Good catch